packages/reporter/cypress/fixtures/runnables.json

```diff
@@ -41,6 +41,35 @@
           ]
         }
       ]
+    },
+    {
+      "id": "r8",
```

Comment:
ℹ️ Additional fixture with different test states to ensure the filters work correctly